JuliaS.txt

```diff
@@ -0,0 +1,3 @@
+Jestem Julia i odkładam wiele rzeczy na ostatnią chwilę D:
```

Comment:
Cześć :) Miło Cię tu widzieć, rozwiąż Jeszce proszę konflikt (możesz na ostatnią chwilę :D chociaż może to być wtedy nieco trudniejsze :D )

Reply:
Cześć! Akurat udało się szybko ogarnąć - czasami się zdarza ;)